src/Compilers/CSharp/Portable/Syntax/InternalSyntax/SyntaxToken.cs

```diff
@@ -84,6 +84,11 @@ public override bool IsToken
             }
         }
 
+        internal bool IsVar
```

Comment:
Consider moving to `SyntaxFacts` as an extension method since it relies on the public API only and since it is specific to actual syntax rather than belonging to the general `SyntaxToken` type.